### DISCUSSION TITLE
Overriding font-size for subheader-line in Big Number Chart on a dashboard causes page crash

### DISCUSSION THREAD

**Discussion by klazaj** [apache/superset#18267] [Q&A / Help]
I tried to override the `subheader-line` class for Big Number chart, by adding this CSS template in 'Edit CSS' option in the dashboard, like so:

```
.subheader-line {
    font-size: 150%!important;
}
```
However, when I click "Save Changes", the page becomes unresponsive. Why is that? How can I override the CSS of Subheader? The font-size becomes too small when I write more than a few words. It's too small even when I choose `Huge` for Font-size option in Chart Options. 

Adding an example for clarity. This is what the Big Number looks like with 'Huge' font-size for the Subheader:
![image](https://user-images.githubusercontent.com/22456202/107016257-04ebfb80-679e-11eb-8064-96d3f20bb2d6.png)
As you can see, it's not really readable. Overriding it improves the readability:
![image](https://user-images.githubusercontent.com/22456202/107016386-2cdb5f00-679e-11eb-8eda-fb589a2dea90.png)

**Comment by russbeye** [comment]
Heads up for anyone else that runs into this: We just experienced this exact issue, and solved it by escaping the `!` character in `!important`.

**Reply by audelorAd** [reply]
Hi,
How do you escape the `!` character in `!important` ?

**Comment by anh-nguyen2021** [comment]
> Heads up for anyone else that runs into this: We just experienced this exact issue, and solved it by escaping the `!` character in `!important`.
I got this issue.  But when you remove the ! character, the font-size doesn't change at all
